docs/EVM/precompiles.md

```diff
@@ -28,10 +28,57 @@ The Frontier EVM used in Astar provides several useful precompiled contracts. Th
 | [Sr25519](precompiles/sr25519.md) | 0x0000000000000000000000000000000000005002     |
 | [SubstrateEcdsa](precompiles/substrate-ecdsa.md) | 0x0000000000000000000000000000000000005003     |
 | [XCM](precompiles/xcm.md) | 0x0000000000000000000000000000000000005004     |
-| assets-erc20 | ASSET_PRECOMPILE_ADDRESS_PREFIX |
-| [XVM](precompiles/xvm.md) | ? |
+| [assets-erc20](precompiles/xc20.md) | ASSET_PRECOMPILE_ADDRESS_PREFIX |
+| [XVM](precompiles/xvm.md) | 0x0000000000000000000000000000000000005005 |
 
 The interface descriptions for these precompiles can be found in the `precompiles` folder: [astar-frame repo](https://github.com/AstarNetwork/astar-frame/).
 
 The Addresses can be checked in the [Astar repo](https://github.com/AstarNetwork/Astar/tree/master/runtime) for each runtime in `precompile.rs` files.
 
+## Third Party Util Precompiles
+
+This is a third-party set of utility libraries that provide helpers functions for writing EVM precompiles. It provides ABI encoding parser and builder, logging utils and execution guards.
+
+Please note that it is neither developed nor supported by Astar Network.
+
+For more information see [the Moonbeam documentation](https://docs.moonbeam.network/builders/pallets-precompiles/precompiles/overview/).
```

Comment:
We shouldn't link this. They've changed their precompiles and it's no longer applicable.
Also, it's not related to builders but to runtime devs.

My suggestion - remove this chapter altogether.

Reply:
By chapter you mean the whole section about utils and their brief description? Or just the link to the Moonbeam docs?

Reply:
The whole section about utils.